.changes/unreleased/Added-20241202-214322.yaml

```diff
@@ -0,0 +1,2 @@
+kind: Added
```

Comment:
Can you add a `body` field with the description here? It helps us generate a changelog

Reply:
Thanks for the review. body-field is added to the yaml-file